migrations/d0ffadc3ea48_add_hidden_columns.py

```diff
@@ -0,0 +1,85 @@
+"""Add hidden columns
+
+Revision ID: d0ffadc3ea48
+Revises:
+Create Date: 2017-09-04 15:19:38.519991
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import flaskbb
+
+
+# revision identifiers, used by Alembic.
+revision = 'd0ffadc3ea48'
+down_revision = None
+branch_labels = ('default',)
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Need mark them nullable first, persist the change, backfill with the proper values, and then mark them non-nullable afterwards.